### ISSUE TITLE
the npy file

### ISSUE THREAD

**Issue by Willamjie** [HuiChen24/IMRAM#1]
Your work is rather good!but I have some question about 'Image features for training set, validation set and testing set should be merged in order into one .npy file',how to merged in order into one file?

**Comment by HuiChen24** [comment]
If you extract image features for by yourself with the pretrained model, you should merge them into one ".npy" file so that the data loader can work appropriately. During the merging, you should guarantee that the image features (i.e. in '*_feat.npy') are placed correctly to be consistent with captions (i.e. in '*_caps.txt'). Please see data.py for more details.

**Comment by Willamjie** [comment]
thank you very much😁------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;HuiChen24&quot;<notifications@github.com&gt;
发送时间:&nbsp;2020年8月3日(星期一) 中午11:13
收件人:&nbsp;&quot;HuiChen24/IMRAM&quot;<IMRAM@noreply.github.com&gt;;
抄送:&nbsp;&quot;jie&quot;<1154311951@qq.com&gt;;&quot;Author&quot;<author@noreply.github.com&gt;;
主题:&nbsp;Re: [HuiChen24/IMRAM] the npy file (#1)
